{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/users/views.py

```diff
@@ -27,6 +29,10 @@ def get_success_url(self):
     def get_object(self):
         return User.objects.get(username=self.request.user.username)
 
+    def form_valid(self, form):
+        messages.add_message(self.request, messages.INFO, _("Infos successfully updated"))
```

Comment:
Looks good, just need to fix Black formatting:

```suggestion
        messages.add_message(
            self.request, messages.INFO, _("Infos successfully updated")
        )
```